internal/worker/sshserver/session.go

```diff
@@ -46,12 +46,12 @@ func (s *sessionHandler) Handle(session ssh.Session, destination virtualhostname
 	}
 
 	switch s.modelType {
-	case state.ModelTypeCAAS:
+	case model.CAAS:
 		if err := s.k8sSessionProxy(session); err != nil {
 			err = errors.Annotate(err, "failed to proxy k8s session")
 			handleError(err)
 		}
-	case state.ModelTypeIAAS:
```

Comment:
I'll keep a note of this, but this is completely wrong. We should never switch on the model type here.